packages/client/src/runtime/utils/serializeRawParameters.ts

```diff
@@ -51,8 +51,7 @@ function encodeParameter(parameter: any, objectSerialization: 'fast' | 'slow'):
   if (isArrayBufferLike(parameter) || ArrayBuffer.isView(parameter)) {
     return {
       prisma__type: 'bytes',
-      // TODO Can we change this now in v16?
-      // TODO: node typings do not include ArrayBufferView as of 14.x
+      // TODO: node typings do not include ArrayBufferView as of 20.x
       prisma__value: Buffer.from(parameter as ArrayBuffer).toString('base64'),
```

Comment:
Should we use `Uint8Array` to avoid explicit usage of `Buffer` here?

Something like:

```ts
function arrayBufferToBase64(arrayBuffer: ArrayBuffer): string {
  const uint8Array = new Uint8Array(arrayBuffer)
  const binaryString = uint8Array.reduce((acc, byte) => acc + String.fromCharCode(byte), '')
  return btoa(binaryString)
}

// Usage
{
  prisma__value: arrayBufferToBase64(parameter as ArrayBuffer)
}
```

Reply:
We have many other places where we use `Buffer` anyway, and the suggested `arrayBufferToBase64` function is too inefficient to be a replacement, although writing a function that directly transforms binary data to base64 is not a problem if we wanted to get rid of the `Buffer`s at some point. I wouldn't change it here in this PR though.